lib/ansible/modules/windows/win_nssm.ps1

```diff
@@ -38,6 +38,12 @@ $stderrFile = Get-AnsibleParam -obj $params -name "stderr_file" -type "path"
 
 $executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
 
+$executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
+$AppRotateBytes = Get-AnsibleParam -obj $params -name "AppRotateBytes" -type "int" -default 104858
```

Comment:
The parameters should follow the `snake_case` format to match other Ansible module options.

Reply:
Other app related params are camel, should I change them as well and add aliases?

Reply:
Yep all parameters should be in snake_case